scripts/calculate-threshold-and-em/calculate_em.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
Remember to set the execution permissions to this file and the other ones. Also the shebang line (and to the other files):

```suggestion
#!/usr/bin/env python
import os
```